src/ESPiLight.h

```diff
@@ -167,8 +167,9 @@ class ESPiLight {
    */
   static const int ERROR_INVALID_PULSETRAIN_MSG_C = -1;
   static const int ERROR_INVALID_PULSETRAIN_MSG_P = -2;
-  static const int ERROR_INVALID_PULSETRAIN_MSG_END = -3;
-  static const int ERROR_INVALID_PULSETRAIN_MSG_TYPE = -4;
+  static const int ERROR_INVALID_PULSETRAIN_MSG_R = -3;
+  static const int ERROR_INVALID_PULSETRAIN_MSG_END = -4;
```

Comment:
Please do not change existing constants, just add a new one.

Reply:
Thought it was better to put it in this order in this case because MSG_R is more similar to MSG_C and MSG_P than the other two. But yeah can move it to the end instead